cmd/storage-rest-server.go

```diff
@@ -670,6 +683,13 @@ func waitForHTTPResponse(respBody io.Reader) (io.Reader, error) {
 			return nil, err
 		}
 		if b != ' ' {
+			if b == 1 {
```

Comment:
is it safe to do something like this?

Reply:
Yes, the legacy stuff (which we wouldn't see anyway since servers are upgraded beyond that) would send spaces. I was actually thinking of completely removing the legacy code since I couldn't think of a case it would apply to. 

Should I do that @harshavardhana ?

Reply:
yes we can do that too but we have to sort of time it with some good feature release, otherwise, users complain about rolling upgrades :-)

Reply:
It will not be a problem for `v17`.

Reply:
All payloads will have a `0x0` before them.

Reply:
Okay